assets/css/style.css

```diff
@@ -88,3 +92,26 @@ li {
 .answers {
   font-family: var(--secondary-font);
 }
+
+/* Footer */
+
+.main {
+  display: flex;
+  flex-direction: column;
+}
+
+.section {
+  flex: 1;
+}
+
+.footer {
+  padding: 2rem 1.5rem 2rem;
```

Comment:
Can these be shortened down to just be top&bottom and left&right as the top padding and bottom padding are both set to 2rem?

Something like:
```
padding: 2rem 1.5rem;
```

Reply:
Good point, updated